### PR TITLE
Use conda on readthedocs, install pandoc and convert notebooks to rst

### PR DESCRIPTION
**Pull request by takluyver**
Alternative to #183 for addressing #156. I'll set up RTD on my fork to test if this works.

### CODE REVIEW

**Comment by Carreau** [comment]
Need rebase. Sorry.

File: docs/conda-env.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: nbconvert_docs
+channels:
+- asmeurer
+dependencies:
+- pandoc
+- nbformat
+- jupyter_client
+- ipykernel
+- numpydoc
```
Comment (minrk) [inline docs/conda-env.yml]: we don't use numpydoc anymore, do we?

Comment (takluyver) [inline docs/conda-env.yml]: I got a failure without this.

**Comment by takluyver** [comment]
Unfortunately, while this is conceptually nicer than #183, it is not currently working - see rtfd/readthedocs.org#1902. As we don't know when that will be resolved, I think we should merge #183 as a stopgap so that our docs are complete.

**Comment by Carreau** [comment]
> Unfortunately, while this is conceptually nicer than #183, it is not currently working - see rtfd/readthedocs.org#1902. As we don't know when that will be resolved, I think we should merge #183 as a stopgap so that our docs are complete.

then we need a travis test that the ipynb->rst conversion is stable, to make sure people don't change the `.rst` file without changing the ipynb.

**Comment by minrk** [comment]
@takluyver makes sense. I merged #183 while we wait for conda to work properly.

**Comment by mgeier** [comment]
There's a possible alternative approach: #212.

**Comment by mgeier** [comment]
I guess this is now obsolete since #212 was merged, right?

**Comment by takluyver** [comment]
I think switching to conda will still make sense, but we no longer need pandoc. I'll open a new PR for it.

**Comment by takluyver** [comment]
Ah, no, we can't easily switch to conda until there's a conda package of nbsphinx. Never mind, requirements.txt works for now.

**Comment by mgeier** [comment]
I have clue about conda packages, but if you tell me how, I can probably make one for `nbsphinx`.
Or feel free to create one yourself ...

**Comment by damianavila** [comment]
If nbsphinx in on pypi it should be easy:

```
conda skeleton pypi nbsphinx
conda builg nbsphinx
anaconda upload -u org path/to/your/conda_package
```

**Comment by mgeier** [comment]
I don't have an anaconda account and I'm not planning to become a `conda` user in the near future. Can somebody else please take care of the package?

And yes, it is on PyPI: https://pypi.python.org/pypi/nbsphinx/

**Comment by minrk** [comment]
nbsphinx doesn't need to be on conda for us to use it. The environment can specify pip dependencies as well (See #222):

```yaml
dependencies:
- pandoc
- ...
- pip:
  - nbsphinx
```

**Pull request closed without merging**